mtest/musicxml/io/testAdditionalFermatas_ref.mscx

```diff
@@ -34,6 +34,11 @@
       <bendFontSize>10</bendFontSize>
       <headerFontSize>10</headerFontSize>
       <footerFontSize>10</footerFontSize>
+      <albumFrontTitleFontSize>10</albumFrontTitleFontSize>
+      <albumFrontComposerFontSize>10</albumFrontComposerFontSize>
+      <albumFrontLyricistFontSize>10</albumFrontLyricistFontSize>
+      <albumContentsTitleFontSize>10</albumContentsTitleFontSize>
+      <albumContentsTextFontSize>10</albumContentsTextFontSize>
```

Comment:
Not sure this is the right fix, default values are not written to score IIRC, so here the default might be wrong?

Reply:
Perhaps those are written unconditionally to the score-file, even if albums feature are not used. This breaks file-version since 3.6.3 might not be able to load an 3.7+Albums score-file.

Reply:
It seems musicxml during import creates all fields unconditionally, resulting in an blown-up mscx.

Reply:
True, I see other settings there too, strangely all with a font size of 10pt.